website/docs/ref/cli.md

```diff
@@ -16,7 +16,7 @@
     {
       "scripts": {
         "extract": "lingui extract",
-        "compile": "lingui compile"
+        "compile": "lingui compile --typescript"
```

Comment:
What if the developer is not using TypeScript? This example is for general use of the CLI, and I'm not sure we need to force adding this flag. I think it's fine to have it below in the full list of available options. What do you think?

Reply:
the idea is to bring visibility to TS support.

Right now, many devs that use TS don't know that there is this option - I didn't notice that option until like 2 months ago 😄 

Running this command will generate a TS file. If people don't use TS, I think they will realize pretty quickly that they just need to remove the `--typescript` flag.

Reply:
Maybe we can add a Docusaurus tip after the code block and mention the `--typescript` parameter? I'm afraid some users might mistakenly think that the `compile` command is only for TypeScript projects or something like that.

Reply:
@andrii-bodnar thanks for the feedback, it's done! :)